modules/viewing-project-metrics-for-distributed-workloads.adoc

```diff
@@ -0,0 +1,71 @@
+:_module-type: PROCEDURE
+
+[id="viewing-project-metrics-for-distributed-workloads_{context}"]
```

Comment:
The correct ID has been used here, but not for the "Viewing distributed workload status" module.